### ISSUE TITLE
/docs/locked_out/ - Resetting a password

### ISSUE THREAD

**Issue by ScuttleSE** [home-assistant/home-assistant.github.io#28544]
### Feedback

> If you are still logged in to the web interface with your user, then you are in luck. Add a new user as an administrator and give the new user a password you can remember. Then log out, and log in with this new user. You may then reset your password via this new administrator account (and then delete this new account), or you can delete your old user account. Either way, your configuration will remain, and you don’t have to do a new onboarding process.

This does not work if you are trying to change password on the initial account you created. 

### URL

https://www.home-assistant.io/docs/locked_out/

### Version

2023.8.2

**Comment by mmseng** [comment]
Yes, this is still an issue in 2023.9.2, Frontend 20230911.0.

The documentation states:

> At the bottom of the dialog box, click “Change Password”. If you do not see the “Change Password” button, try using another interface, such as the mobile version.

However I've tried logging in with 5 different browsers (including mobile) and the "Change Password" button does not show up for me. And yes, I enabled "Advanced mode" on the new admin account I created.

This was submitted and ignored in the following issue as well: https://github.com/home-assistant/home-assistant.io/issues/23048

Thankfully in my case I was able to use the instructions for resetting the password via the container commandline.

**Comment by reedy** [comment]
It is kinda documented at https://www.home-assistant.io/docs/locked_out/#start-a-new-onboarding-process but it should be clarified/mentioned further up

From #23048

>The instructions to change a user's password do not apply to recent versions of HA. Only the owner account can now change another user's password, not an administrator account

I'd be curious what the "recent version" of HA is/was... Because this feels like quite a big documentation hole that is relatively simply fixed, especially for someone that knows what the matrix of who can reset what, when etc...

**Comment by Judman** [comment]
> Yes, this is still an issue in 2023.9.2, Frontend 20230911.0.
> 
> The documentation states:
> 
> > At the bottom of the dialog box, click “Change Password”. If you do not see the “Change Password” button, try using another interface, such as the mobile version.
> 
> However I've tried logging in with 5 different browsers (including mobile) and the "Change Password" button does not show up for me. And yes, I enabled "Advanced mode" on the new admin account I created.
> 
> This was submitted and ignored in the following issue as well: #23048
> 
> Thankfully in my case I was able to use the instructions for resetting the password via the container commandline.

Yep! I understand this is a big heap of software to document, but user management is huge and is changed are made to it then those changed need to be documented. Especially since that page is the first google result a lot of people will see.
